packages/common-all/src/store/ConfigStore.ts

```diff
@@ -61,12 +70,34 @@ export class ConfigStore implements IConfigStore {
     }
   }
 
-  writeConfig(payload: DendronConfig) {
+  writeConfig(wsRoot: URI, payload: DendronConfig) {
     return YamlUtils.toStr(payload)
-      .asyncAndThen((endPayload) => this.writeToFS(this.configPath, endPayload))
+      .asyncAndThen((endPayload) =>
+        this.writeToFS(this.configPath(wsRoot), endPayload)
+      )
       .map(() => payload);
   }
 
+  writeOverride(
+    wsRoot: URI,
+    config: DeepPartial<DendronConfig>,
+    mode: "workspace" | "global"
+  ) {
+    const res = YamlUtils.toStr(config).map((content) => {
+      const writePath = this.configOverridePath(wsRoot, mode);
+      if (writePath) {
+        return this.writeToFS(writePath, content);
+      } else {
+        return errAsync(
+          new DendronError({
+            message: "global override not supported with current file store.",
+          })
+        );
+      }
+    });
+    return res.unwrapOr(() => {});
```

Comment:
Doing this here to unwrap a `Result<ResultAsync<...>>` to a `ResultAsync<>`...

is there a better way to handle this? maybe @namjul knows :)